CMakeLists.txt

```diff
@@ -52,6 +52,8 @@ endif()
 # Settings
 #
 include(CcacheVersion)
+include(UseCcache)
+include(UseFastestLinker)
```

Comment:
Since the `if` block below is associated with `include(CcacheVersion)` above I think it makes sense to move these two lines below the `if` block.